_pages/en/strategy-build-vs-buy.md

```diff
@@ -224,14 +314,14 @@ For details as to why the current approach to build vs buy decisions is out of a
 
 For further details regarding the increase of build decisions (based on the new definition of build, as defined in the Definitions section), reference Annex 3: Alignment.
 
-##### 3. Use Service Oriented Architecture (SOA)
+##### 3. Use Service Oriented Architecture (SOA)<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).

This is also a repeat of GC Architecture Standards